build/static/boot/functions.sh

```diff
@@ -1,3 +1,5 @@
+#!/usr/bin/env bash
+
```

Comment:
This file is not executable so I don't know if it need shebang. I really don't know.